docs/dev/fundamentals/rollups.md

```diff
@@ -0,0 +1,37 @@
+# Introduction
+
+To better understand rollups we need to dive briefly into Ethereum and Layer 2 solutions.
+
+The enormous demands from users to transact Ethereum put a limit on the network capacity by causing congestion, resulting in slow transactions and increased gas prices. While this has remained so for a while, an improved solution is needed, one that can maintain scalability.
```

Comment:
```suggestion
The enormous demand from users to transact on Ethereum puts a limit on the network capacity by causing congestion, resulting in slow transactions and increased gas prices. While this has remained so for a while, an improved solution is needed, one that can maintain scalability.

Reply:
> puts a limit on the network capacity

> resulting in slow transactions

I'd say that network capacity and "slowness" are part of the design, not the actual usage

Reply:
@krofax why have you marked this question as resolved without resolving the issues raised by @barakshani?